lbrynet/reflector/client/client.py

```diff
@@ -407,6 +407,10 @@ def send_blob_info(self):
             'blob_size': self.next_blob_to_send.length
         }))
 
+    def log_fail_and_disconnect(self, err, blob_hash):
+        log.error("Error reflecting blob %s", blob_hash)
+        self.transport.loseConnection()
```

Comment:
could have sworn I replied to this - I think losing the connection is fine.  Maybe we'll want to have the caller retry at some point?